package-lock.json

```diff
@@ -0,0 +1,3717 @@
+{
```

Comment:
@thfontaine Please remove `package-lock.json` from git. The repository is using `yarn` and `yarn.lock` for package management.